### ISSUE TITLE
html 根节点语言标记应匹配内文

### ISSUE THREAD

**Issue by Artoria2e5** [orvice/ss-panel#551]
Version: HEAD

ss-panel 很多输出 HTML 的语言标记都是 `en`，劣化了 Google Chrome 浏览器的字体选择。举个例子吧，[所有含“用户中心”四个字的页面](https://github.com/orvice/ss-panel/search?utf8=%E2%9C%93&q=用户中心)里面，很多都是纯中文的，而其 lang 标记都是 `en`。

（当然，不标在根节点上，标在文字出现的各个 tag 上都是可以的。）

---

顺便问一下，这东西这样硬编码是没有搞 i18n 的意思了咯……？（PHP 是有[gettext](https://secure.php.net/manual/en/book.gettext.php)的，要搞的话不要造什么复数吃瘪的大词典方轮子……）

---

``` JS
document.body.lang = "zh-Hans-CN"
```

UserScript 绝赞套用中

**Comment by Artoria2e5** [comment]
Hi comrade, you didn't fix this.

**Comment by Artoria2e5** [comment]
A bunch of your views templates are written with a Hardcoded en.

**Comment by orvice** [comment]
Hi, I'm not good at js, may i fix is this way ' document.body.lang = navigator.language || navigator.userLanguage; '? 

PS: i18n has been suppported: https://sspanel.xyz/docs/dev/i18n

**Comment by Artoria2e5** [comment]
Nope, matching it with the user's browser language won't help at all. Match it with your text language -- that's whatever you are setting in your i18n thing. Oh and `zh-Hans-CN` instead of `zh_cn` if you want to be pedantic.

**Comment by Artoria2e5** [comment]
If your PHP part knows what language the user is seeing, you don't even need any js. 
```
<html lang="<?=whatever_language_variable?>">
```
That's it.
